DrawerLayout.js

```diff
@@ -507,7 +507,7 @@ export default class DrawerLayout extends Component<PropType, StateType> {
 }
 
 const styles = StyleSheet.create({
-  drawer: { flex: 0 },
+  drawer: {},
```

Comment:
why?

Reply:
With `flex: 0` the `Drawer` will not be visible on web and though I removed it it was working the same on native. Did you notice any difference or why did you choose `flex: 0`?

Reply:
Let's remove `styles.drawer` than altogether, what's the point of having an empty style?

Reply:
Removed in https://github.com/kmagiera/react-native-gesture-handler/pull/406/commits/23af89f77c0620cf3b20372d8df0636172793f7a